### ISSUE TITLE
Tests for twisted.conch should conform to the new style of naming test classes

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7698]
|[<img alt="eeshangarg's avatar" src="https://avatars.githubusercontent.com/u/7251823?s=50" width="50" height="50">](https://github.com/eeshangarg)| @eeshangarg reported|
|-|-|
|Trac ID|trac#7698|
|Type|enhancement|
|Created|2014-10-29 16:30:05Z|
|Branch|https://github.com/twisted/twisted/tree/conch-tests-names-7698|

This ticket is a follow-up to ticket [ticket:7032].

Attachments:

* [bugfix_7698.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/955/9554f2e1fb91b8a4baaf660fc2888900a296bf37/e96a049100c1e9ffd25eaa1a565487c603d79ae9.patch) (25246 bytes) - added by eeshangarg on 2014-10-29 16:55:41Z - 
* [bugfix_7698_2.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/955/9554f2e1fb91b8a4baaf660fc2888900a296bf37/2b0aa540e6a28faad4e5d6050ff313a25bbc4a31.patch) (25354 bytes) - added by eeshangarg on 2014-11-19 11:38:40Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__7698 7698
type__enhancement enhancement
reporter__eeshangarg eeshangarg
priority__normal normal
milestone__ 
branch__branches_conch_tests_names_7698 branches/conch-tests-names-7698
branch_author__eeshangarg eeshangarg
status__closed closed
resolution__fixed fixed
component__conch conch
keywords__easy easy
time__1414600205466254 1414600205466254
changetime__1417022163201172 1417022163201172
version__None None
owner__exarkun exarkun
cc__eeshangarg cc__z3p
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="eeshangarg's avatar" src="https://avatars.githubusercontent.com/u/7251823?s=50" width="50" height="50">](https://github.com/eeshangarg)<a name="note_2"></a>|@eeshangarg set owner to @glyph|
|-|-|

Hi! I just added a patch that changes all of the tests in twisted.conch to use the new style of naming test classes as discussed in ticket [ticket:7032].

Thanks!
Eeshan

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_3"></a>|@glyph removed owner|
|-|-|

Un-assigning all review tickets assigned to me.  Unless there's a specific reason _I_ need to review something, let's leave them unassigned :-).

**Comment by twisted-trac** [comment]
|[<img alt="eeshangarg's avatar" src="https://avatars.githubusercontent.com/u/7251823?s=50" width="50" height="50">](https://github.com/eeshangarg)<a name="note_4"></a>|@eeshangarg commented|
|-|-|

Hi!

I just submitted a new patch for this called bugfix_7698_2.patch, which includes a .misc newsfile along with the contents of the previous patch. :)

Thanks & regards,
Eeshan Garg

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun set owner to @exarkun|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_6"></a>|@exarkun commented|
|-|-|

(In [43499]) Branching to 'conch-tests-names-7698'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_7"></a>|@exarkun commented|
|-|-|

(In [43501]) Apply bugfix_7698_2.patch 

Refs [#7698](https://github.com/twisted/twisted/issues/7698)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_8"></a>|@exarkun commented|
|-|-|

Thanks.  These changes all look fine to me. Buildbot, apart from twistedchecker and some other unrelated failures, seems happy (and twistedchecker is just confused by the new names). I'll merge.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_9"></a>|@exarkun set status to `closed`|
|-|-|

(In [43502]) Merge conch-tests-names-7698

Author: eeshangarg
Reviewer: exarkun
Fixes: [#7698](https://github.com/twisted/twisted/issues/7698)

Rename all of the test case classes in Twisted Conch to conform to the
recently documented test case class naming convention.
